molnsutil/molnsutil.py

```diff
@@ -854,6 +881,9 @@ def __init__(self, model_class, parameters, client=None, num_engines=None):
               e.g.: {'arg1':[1,2,3],'arg2':[1,2,3]}  will produce 9 parameter points.
             If it is a list, where each element of the list is a dict
             """
+
+#	DistributedEnsemble.__init__(self, model_class, parameters, client, num_engines)
```

Comment:
Wait, no, it should not be commented. But this is part of fixing another bug, still working on it. This pull is about fixing the set up of the paramter space so that it creates the correct number of parameters.